onnxruntime/test/perftest/command_args_parser.cc

```diff
@@ -46,12 +46,13 @@ namespace perftest {
       "\t-P: Use parallel executor instead of sequential executor.\n"
       "\t-o [optimization level]: Default is 1. Valid values are 0 (disable), 1 (basic), 2 (extended), 99 (all).\n"
       "\t\tPlease see onnxruntime_c_api.h (enum GraphOptimizationLevel) for the full list of all optimization levels. \n"
+      "\t-u [optimized_model_path]: Specify the optimized model path for saving.\n"
```

Comment:
This is useful. I hope that shape inference result could be also be output in the graph (maybe in another pull request).

How about change -u to -O? Since **O**ptimized is easy to remember (The drawback is -o is taken).

Reply:
I thought -O might be reserved for other more frequent options. This option is relatively infrequent so I picked u to avoid conflictions.

---
In reply to: [338309668](https://github.com/microsoft/onnxruntime/pull/2227#discussion_r338309668) [](ancestors = 338309668)